src/main/resources/application.properties

```diff
@@ -28,12 +28,9 @@ quarkus.helm.dependencies.postgresql.alias=db
 quarkus.helm.dependencies.postgresql.version=11.9.1
 quarkus.helm.dependencies.postgresql.repository=https://charts.bitnami.com/bitnami
 quarkus.helm.dependencies.postgresql.condition=db.enabled
-quarkus.helm.values."db.enabled".value-as-bool=true
 ## This configuration must match with the configuration in `src/main/helm/values.yaml`
-%prod.quarkus.datasource.username=user
-%prod.quarkus.datasource.password=pwd
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://${DB_SERVICE_NAME:fruits-app-db}:5432/fruits_database
+%prod.quarkus.datasource.username=${db.auth.username}
+%prod.quarkus.datasource.password=${db.auth.password}
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://${DB_SERVICE_NAME:fruits-app-db}:5432/${db.auth.database}
```

Comment:
Is it documented that the properties `db.auth.xxx` must be defined using the helm values when application is started using `prod` profile ?

Reply:
It's not mandatory that the properties `db.auth.*` are defined in the Helm values. 
If they are not, the Quarkus Helm extension will generate an Env Var property to map this configuration. 
But if they are defined in the Helm values file, the Quarkus Helm extension will link these properties automatically, so the values provided to Helm will always be synced with the configuration that the Quarkus reads from the application properties.